Minor Arrangement Edit

jamesche75 opened the pull request:
Moved the section about how to open App Service Diagnostics to the top of the article instead of the bottom.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

jamesche75 committed:
Minor Arrangement Edit

Moved the section about how to open App Service Diagnostics to the top of the article instead of the bottom.
